pkg/subcmd/config.go

```diff
@@ -169,6 +177,32 @@ func (c *Config) runCreate() error {
 	return err
 }
 
+// runDelete controls the deletion process.
+func (c *Config) runDelete() error {
+	if c.flags.DryRun {
+		c.log().Debug("[DRY-RUN] Configuration is not removed from the cluster")
+		return nil
+	}
+	return c.manager.Delete(c.cmd.Context())
+}
+
+// runGet controls the cluster configuration retrieval process.
+func (c *Config) runGet() error {
+	c.log().Debug("Retrieving the cluster configuration")
+	cfg, err := c.manager.GetConfig(c.cmd.Context())
+	if err != nil {
+		if c.create && c.flags.DryRun {
+			c.log().Warn(
+				"[DRY-RUN] Configuration does not exist in the cluster, yet.")
```

Comment:
Same comment here: I would just print out what it would try to do if dry run is true

Reply:
I'm not sure `--dry-run` is relevant on a `get`, since it's a read-only operation.

Reply:
Well, the tool supports using `--get` and `--create` at the same time, and that's something I use all the time. Otherwise, `--create --get --dry-run` would return error, something we should avoid when `--dry-run` is informed.